ci/cron/src/Main.hs

```diff
@@ -344,6 +344,7 @@ verify_signatures :: FilePath -> FilePath -> String -> IO ()
 verify_signatures bash_lib tmp version_tag = do
     System.callCommand $ unlines ["bash -c '",
         "set -euo pipefail",
+        "export DADE_SKIP_JAVA=1",
```

Comment:
instead of trying to get this right everywhere in here, can we just call `dade-assist` once outside of the script and then skip it? I don’t understand why we need to call it separately for each of those invocations.

Reply:
Actually, we already do call `dade-assist` before running this. 🤔